frontend/src/static/js/v_authorship.js

```diff
@@ -15,6 +15,15 @@ function toggleNext(ele) {
   parent.className = classes.join(' ');
 };
 
+function expandAll(isActive) {
+  const renameValue = isActive ? 'file active' : 'file';
+
+  const files = document.getElementsByClassName('file');
+  Array.from(files).forEach((file) => {
```

Comment:
i think you can just use `files.forEach` without the `Array.from`, if i ain't wrong.